browser-test/src/admin_program_creation.test.ts

```diff
@@ -63,58 +65,78 @@ describe('program creation', () => {
     await adminProgramImage.expectProgramImagePage()
   })
 
-  it('program details page screenshot', async () => {
-    const {page, adminPrograms} = ctx
-
+  it('create program then go back prevents name edits', async () => {
```

Comment:
suggestion (non-blocking): Maybe call this "prevents url edits"? It's confusing that "name" actually refers to the applicant facing url.